scripts/build-wasm.rb

```diff
@@ -48,7 +48,7 @@ def self.debug(verbose: false)
         #
         # - https://github.com/zackradisic/cheatsheets/issues/21#issuecomment-969393554
         # - https://github.com/rust-lang/rust/issues/85821#issuecomment-969369677
-        ENV['EMMAKEN_CFLAGS'] = '-s ERROR_ON_UNDEFINED_SYMBOLS=0 --no-entry'
+        ENV['EMCC_CFLAGS'] = '-s ERROR_ON_UNDEFINED_SYMBOLS=0 --no-entry'
```

Comment:
thanks for making this change!

😅 cargo I think masks these warnings when the build succeeds. Thanks for noticing this in the build output.

Linking the ticket from upstream that made the deprecation:

- https://github.com/emscripten-core/emscripten/issues/15684
- https://github.com/emscripten-core/emscripten/pull/15685

Reply:
yeah and I saw `EMCC_CFLAGS` added since 2012 https://github.com/emscripten-core/emscripten/commit/40f2ae9e943a66cd8a3fa2ef1375fb9d81fcd3bd

So I think here's change is safe